gyp/pylib/gyp/msvs_emulation.py

```diff
@@ -237,6 +237,11 @@ def GetExtension(self):
   def GetVSMacroEnv(self, base_to_build=None, config=None):
     """Get a dict of variables mapping internal VS macro names to their gyp
     equivalents."""
+    target_arch = self.GetArch(config)
+    if target_arch == 'x86':
+      target_platform = 'Win32'
+    else:
+      target_platform = target_arch
     target_platform = 'Win32' if self.GetArch(config) == 'x86' else 'x64'
```

Comment:
I believe this addition replaces the line following it, so line 245 here should be deleted.